cmd/state-svc/service.go

```diff
@@ -82,3 +92,32 @@ func (s *service) Wait() error {
 	}
 	return nil
 }
+
+type resolveExec struct {
+	resolver *resolver.Resolver
+}
+
+func (r *resolveExec) ReportRuntimeUsage(ctx context.Context, pid, exec string) {
+	dims := &dimensions.Values{
+		Trigger:          p.StrP(target.TriggerExec.String()),
+		Headless:         p.StrP("true"),
```

Comment:
This is expressly for the `state-exec` use case, so I think Headless will always be true. Right? Requiring the commit id and namespace will induce more work, that, I think, will necessitate the executors having that data printed into them when they are generated. Sorry if I'm misunderstanding. I'm not sure how else the commit id and namespace would be acquired otherwise, but I'm open to options.

Reply:
Oh, hmm, is that headless commit? I'm not even sure what I had it in mind as. If so, that data will need to be included in the executor templates as well.

Reply:
Headless has no bearing on executors. Headless just means what state your local project is in, and executors follow the state of your local project. It can be true or false.